### ISSUE TITLE
Tracking issue

### ISSUE THREAD

**Issue by mjlbach** [StanfordVL/OmniGibson#2]
- [ ] many methods define a wrapper (`step()` that trivially calls `_step()`). This type of abstraction can be valuable but adds cognitive overhead, and since all of these wrapper functions don't currently do anything I think we should remove
- [ ] `Registerable` is a little hard to follow, it would be easier to just use a registration function
- [ ] Too many nested inheritance chains split function definitions across several files
- [ ] Missing config attribute in termination/reward functions (may or may not be necessary, but I personally miss
- [ ] switch to omegaconf
- [ ] missing USD conversion script for iG objects
- [ ] Dummy task has default_x and _create termination condition?
- [ ] Make rendering settings configurable per sensor
- [ ] rendering artifact with pathtracing
- [ ] Fix performance regression with contact sensors
- [ ] Add env.render() for gym compatibility

**Comment by cremebrule** [comment]
closing since ~1 year ago, most of this is outdated
